test/integration/rules/aria-allowed-attr/passes.html

```diff
@@ -1941,3 +1941,7 @@
     </tr>
   </tbody>
 </table>
+
+<div role="comment" aria-level="1" id="pass91">ok</div>
+<div role="comment" aria-posinset="1" id="pass92">ok</div>
+<div role="comment" aria-setsize="16" id="pass93">ok</div>
```

Comment:
For this particular test we can do this in just a single element since if any one of the attributes is not allow the element will fail.

```suggestion
<div 
  id="pass91"
  role="comment" 
  aria-level="1" 
  aria-posinset="1" 
  aria-setsize="16" 
>ok</div>
```